### PR TITLE
Tempo: Update docs for trace to logs functionality

### PR DESCRIPTION
**Pull request by aocenas**
See https://github.com/grafana/grafana/pull/61702 for the feature changes.

PR with new image https://github.com/grafana/website/pull/11702

### CODE REVIEW

File: docs/sources/datasources/tempo/_index.md
Comment (joey-grafana) [inline docs/sources/datasources/tempo/_index.md:58]: Might not be totally clear what template language means in this case

File: docs/sources/datasources/tempo/_index.md
Comment (joey-grafana) [inline docs/sources/datasources/tempo/_index.md:71]: `even those not mapped here` is a bit unclear

File: docs/sources/datasources/tempo/_index.md
Comment (joey-grafana) [inline docs/sources/datasources/tempo/_index.md:89]: Only some of these descriptions have a full stop

**Review by joey-grafana — APPROVED** [review]
Technically I think this (mostly) makes sense apart from a few comments.

**Comment by joey-grafana** [comment]
Docs review will he helpful (as always before merging) of course.

File: docs/sources/datasources/tempo/_index.md
Comment (CrypticSignal) [inline docs/sources/datasources/tempo/_index.md:70]:
```suggestion
1. Set start and end time shift. As the logs timestamps may not exactly match the timestamps of the spans in the trace it may be necessary to widen or shift the time range to find the desired logs.
```

File: docs/sources/datasources/tempo/_index.md
Comment (CrypticSignal) [inline docs/sources/datasources/tempo/_index.md:71]:
```suggestion
1. Optionally select tags to map. In the custom query you can use all tags separately, even those not mapped here or you can use ${\_\_tags} variable which will use and interpolate the tags mapped here. The tags you configure must be present in the span's attributes or resources for a trace to logs span link to appear. You can optionally configure a new name for the tag. This is useful in cases where the tag has dots in the name and the target data source does not allow using dots in labels. For example, you can remap `http.status` to `http_status` in such a case.
```

**Comment by joey-grafana** [comment]
@CrypticSignal when there is so much altered text in the docs it's usually best to leave it to the experts!

File: docs/sources/datasources/tempo/_index.md
Comment (aocenas) [reply docs/sources/datasources/tempo/_index.md:58]: Added a link to the variable docs

Comment (joey-grafana) [reply docs/sources/datasources/tempo/_index.md]: Looks good

File: docs/sources/datasources/tempo/_index.md
Comment (aocenas) [reply docs/sources/datasources/tempo/_index.md]: I reworded the description here, please let me know if it is clearer like this.

Comment (joey-grafana) [reply docs/sources/datasources/tempo/_index.md]: Thanks. This is much clearer.

**Pull request merged**
